skrub/_dataframe/tests/test_common.py

```diff
@@ -0,0 +1,147 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_array_equal
+
+from skrub._dataframe import _common as ns
+
+#
+# Inspecting containers' type and module
+# ======================================
+#
+
+
+def test_skrub_namespace(df_module):
```

Comment:
I think that it would be worth to provide the location in a module docstring of this feature.

Reply:
> you mean state in test_common.__doc__ that df_module is a pytest fixture defined in conftest.py?

Just on the top of the file, adding a multiline docstring with the info that you mentioned. "module docstring" was a bit vague.